features/admin.users.v1/components/user-change-password.tsx

```diff
@@ -657,7 +680,9 @@ export const ChangePasswordComponent: FunctionComponent<ChangePasswordPropsInter
                         <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 14 }>
                             <Message
                                 type="warning"
-                                content={ t("user:modals.changePasswordModal.message") }
+                                content={ isResetPassword
+                                    ? t("user:modals.changePasswordModal.message")
+                                    :  t("user:modals.setPasswordModal.message") }
```

Comment:
remove this code

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/399e7573e645d5c76ea0e9c6b3a47780a2c40019 .